Misc/NEWS.d/next/C_API/2024-11-29-18-33-27.gh-issue-74695.xXInJz.rst

```diff
@@ -0,0 +1,3 @@
+Added support for ``ctypes.c_bool`` for structs/unions that have an endianness opposite of the current system.
+
+Previously, on a little endian system, attempting to use a ``c_bool`` field in a ``BigEndianStructure`` would result in an exception (TypeError: This type does not support other endian: <class 'ctypes.c_bool'>). Now, it works as expected.

```

Comment:
Love the effort on the blurb entry, but it doesn't need to be nearly this long. I think just the first sentence is fine.
```suggestion
```